caa/templates/konshuu.html

```diff
@@ -0,0 +1,95 @@
+{% extends "base.html" %}
+
+{% block title %}Konshuu{% endblock %}
+
+{% block content %}
+<h1>Konshuu</h1>
+
+<p>Konshuu is Cal Animage Alpha's weekly newsletter, featuring articles and fanworks from both guests and current officers. Each issue includes an overview of the series that will be shown/read during weekly meetings, information about future events, and an assortment of articles on anime culture. The latest issue is available for free at each weekly showing.</p>
+
+<h2> Contributing to Konshuu</h2>
+
+<p>
+Guest submissions are always welcome! To contribute, visit <a target="_blank" href="http://j.mp/konsub">j.mp/konsub</a>.</p>
+
+<p>If you're interested in becoming a regular contributor to the newsletter, please contact the current head—Tina (mechato on IRC)—about working in the Konshuu department</p>
+
+<!--<h2>Digital Archive</h2>
```

Comment:
You can delete these lines instead of commenting it out. Praise version control!